backend/app/main.py

```
@@ -1 +1,127 @@
+        status="healthy" if db_healthy and llm_healthy else "unhealthy",
+        timestamp=datetime.now(),
+        version=__version__,
+        services={
+            "database": db_healthy,
+            "llm": llm_healthy,
+        }
+    )
+
+
+# APIルーター登録
+from app.api import create_tables, generate_problem, check_answer, table_schemas
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)